### PR TITLE
Add form to add observing run on observing run page

### PR DESCRIPTION
**Pull request by dannygoldstein**
[ch1240]

This PR adds a form that allows users to add a new observing run on the observing run page.

### CODE REVIEW

**Review by kmshin1397 — COMMENTED** [review]
LGTM! One suggestion - I think it would be good to add a notification for successful submits (like we do in ClassificationForm.jsx). I know it'll shows up in the list next to the form but I was still unsure if it worked at first because I wasn't focused on the list.

**Comment by dannygoldstein** [comment]
@kmshin1397 Done

File: static/js/components/ObservingRunPage.jsx
Comment (dmitryduev) [inline static/js/components/ObservingRunPage.jsx:57]: Why do you need both xs and sm?

File: static/js/components/ObservingRunPage.jsx
Comment (dmitryduev) [inline static/js/components/ObservingRunPage.jsx:65]: And here?

**Review by dmitryduev — APPROVED** [review]
All looks good! One comment is that we should probably add the ability to edit the existing runs in the future.

**Comment by kmshin1397** [comment]
@dannygoldstein I don't see any new changes here - did you forget to push?

**Comment by dannygoldstein** [comment]
@kmshin1397 Yeah, nice catch, thanks. Pushed.

**Review by kmshin1397 — APPROVED** [review]

**Comment by dmitryduev** [comment]
@dannygoldstein Are there any show stoppers for merging here?

**Comment by dannygoldstein** [comment]
Yes, dont merge yet. Something weird is happenign with the tests.

File: static/js/components/ObservingRunPage.jsx
Comment (dannygoldstein) [reply static/js/components/ObservingRunPage.jsx:57]: fixed

File: static/js/components/ObservingRunPage.jsx
Comment (dannygoldstein) [reply static/js/components/ObservingRunPage.jsx:65]: fixed

**Comment by dannygoldstein** [comment]
This PR also fixes this issue raised by Dan Perley:

I did notice that the list of observing runs comes up surprisingly slowly (takes about 8 seconds) given the efficiency of the website everywhere else.

**Comment by dannygoldstein** [comment]
Failures unrelated

**Review by kmshin1397 — APPROVED** [review]
LGTM! Just gonna rerun Travis to see if it gets any better, seeing as there are some shared tests failing even though they are unrelated. I'll probably merge regardless once that's done.

**Comment by kmshin1397** [comment]
Looks like @dmitryduev will need to take a quick look again - I think I lost write access to the repo. Also the weather widget test seems to be failing consistently. We might want to look into making that more stable at some point.

**Review by stefanv — APPROVED** [review]

**Comment by yaoyuhan** [comment]
Can someone please merge this? Need to be able to add observing run to Fritz. Thanks!

**Comment by acrellin** [comment]
@yaoyuhan this has been merged, and we are working on updating the current deployment. It is a process and requires more than someone pushing a button. Thanks for your understanding and patience.

**Comment by yaoyuhan** [comment]
Thanks @acrellin ! Our next ZTF runs are Oct 20 and Oct 21, and it would be really nice to have the button ready before those runs, so that we can use Fritz to test the startlist, etc. Thanks! --Yuhan

**Pull request merged**
